build.ps1

```diff
@@ -24,14 +21,18 @@ Write-Output "Using MSBuild from: $msbuild"
 if ($lastexitcode -ne 0) { exit $lastexitcode; }
 
 # Create the stable NuGet package
-& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix=".$env:BUILD_NUMBER"
+echo "Creating package"
+& $msbuild "./Xamarin.Essentials/Xamarin.Essentials.csproj" /t:Pack /p:Configuration=Release /p:ContinuousIntegrationBuild=$cibuild /p:Deterministic=false /p:VersionSuffix=".$date-git$hash"
```

Comment:
can you tweak the version to include hour:minute too? I want it to be the same format as this sample: 0.10.0-date20191005-0857.git-c03b206 (see https://www.nuget.org/packages/Xamarin.Forms.Background )

Reply:
Done